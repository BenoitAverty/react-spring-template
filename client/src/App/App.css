.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-menu {
  background-color: #5FB3B3;
  height: 3em;
  margin: 0;
}

.App-menu ul {
  display: flex;
  flex-direction: row;
  padding: 0;
  margin: 0;
  justify-content: center;
  align-items: center;
  height: 100%;
}

.App-menu li {
  list-style-type: none;
  height: 100%;
  margin: 0 10px;
}

.App-menu a {
  display: flex;
  height: 100%;
  padding: 0 20px;
  align-items: center;
  color: inherit;
  text-decoration: none;
  font-weight: bold;
}

.App-intro {
  font-size: large;
}

.App-content {
  margin: 2em auto;
  width: 70%;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}
