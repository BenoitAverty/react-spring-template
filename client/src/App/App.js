import React from 'react'

import {Link} from 'react-router'

import logo from './logo.svg'
import './App.css'

const App = (props) => {
  return (
      <div className="App">
        <div className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h2>Welcome to React</h2>
        </div>
        <div className="App-menu">
          <ul>
            <li><Link to="/">Main</Link></li>
            <li><Link to="/login">Login</Link></li>
          </ul>
        </div>
        <div className="App-content">
          {props.children}
        </div>
      </div>
  );
}

export default App;
