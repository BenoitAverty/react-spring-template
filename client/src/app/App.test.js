import React from 'react';

import App from './App';

it('Matches latest snapshot', () => {
  const element = render(<App />)

  expect(element).toMatchSnapshot()
});
