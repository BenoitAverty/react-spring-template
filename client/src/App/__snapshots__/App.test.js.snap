exports[`test Matches latest snapshot 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <img
      alt="logo"
      className="App-logo"
      src="test-file-stub" />
    <h2>
      Welcome to React
    </h2>
  </div>
  <div
    className="App-menu">
    <Nav
      bsClass="nav"
      bsStyle="tabs"
      justified={false}
      pullLeft={false}
      pullRight={false}
      stacked={false}>
      <IndexLinkContainer
        to="/">
        <NavItem
          active={false}
          disabled={false}>
          Home
        </NavItem>
      </IndexLinkContainer>
      <LinkContainer
        action="push"
        onlyActiveOnIndex={false}
        to="/login">
        <NavItem
          active={false}
          disabled={false}>
          Login
        </NavItem>
      </LinkContainer>
    </Nav>
  </div>
  <div
    className="App-content" />
</div>
`;
